CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 - This release supports OneView Rest API versions 800/1000/1200 minimally where we can use OneView v4.10/v4.20/v5.0 with this SDK. No new fields are added/deleted to support OneView Rest API 800/1000/1200. Complete support will be done in next releases.
 - Extending support to API 600 with the following HPE OneView resource:
 - oneview_scope
+- oneview_volume_template
+- oneview_volume
+- oneview_storage_pool
+- oneview_storage_system
 
 ### Oneview Features supported
 - oneview_connection_template
```

Comment:
Add the supported resources here in alphabetical order.

Reply:
corrected